docs/integrations/duckdb.md

```diff
@@ -72,3 +73,19 @@ USING SAMPLE reservoir(50000 ROWS) REPEATABLE (100);
 
 COPY sampled_population TO 's3://example-repo/main/data/population/sample.parquet'; -- actual write happens here
 ```
+
+## Using DuckDB in the lakeFS Web Interface
```

Comment:
Usually in the docs we reference it as UI or web UI I think

Reply:
thanks, fixed.